leaf-ui/js/object/evoObjects.js

```diff
@@ -631,7 +739,20 @@ class EVONextState {
         if (EVONextState.isColorBlindMode) {
             return EVO.colorVisDictColorBlind[intentionEval];
         }
-        return EVO.colorVisDict[intentionEval]; }
+
+        if (EVONextState.paletteOption < 5) {
+
+            return EVO.colorVisDictCollection[EVONextState.paletteOption - 1][intentionEval];
+        }
+        if (EVONextState.paletteOption == 5) {
+            var selfVis = myInputJSObject.results.get('colorVis').selfColorVisDict;
+
+            return selfVis[intentionEval];
+        }
+
+    }
+
+        //return EVO.colorVisDict[intentionEval]; }
```

Comment:
Are you saving this line for something later?

Reply:
just removed!